env.js

```diff
@@ -1,13 +1,14 @@
 var log = require('loglevel');
 
 module.exports = function() {
-  if (process.env.COUCH2PG_DEBUG) {
+  if (process.env.COUCH2PG_DEBUG || true) {
     log.setDefaultLevel('debug');
   } else {
     log.setDefaultLevel('info');
   }
 
   return {
+    legacyMode: process.env.LEGACY_MODE || false,
```

Comment:
So the intention here is that 0.4 is "legacy", and everything else, 2.6 and onward, is considered current. Thinking about it in the fresh light of day, perhaps this doesn't fit with Medic's deployment practices particularly well.

Currently the only difference is that if you have legacy mode turned on certain database migrations to add views get added, as those views are not relevant to 0.4.

I'll have a think about ways to make this clearly / feature toggled. In terms of detecting it we could possibly intentionally download the ddoc and get the version from that. Not sure we need that much structure yet though.